tests/saved_test_data/sample_relion_data.star

```diff
@@ -47,3 +47,4 @@ _rlnRandomSubset #29
   757.018043  2660.755621            1     0.932745   -92.812197 000015@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   134.333245    92.320564    -0.300400     0.199600     0.706685 1.426995e+05     0.074965           42            1
  3165.259343  2881.675032            1     0.513467    97.907345 000016@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   128.513119    60.205933     0.449600    -0.050400     0.703292 1.428023e+05     0.094331           33            1
   916.590156  3057.649585            1     0.601333  -120.014857 000017@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   130.487578    87.229432     0.449600    -0.050400     0.706048 1.425358e+05     0.126520           26            2
+  916.590156  3057.649585            1     0.601333  -120.014857 000001@sample_one_particle.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375	   7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   130.487578	  87.229432     0.449600    -0.050400     0.706048 1.425358e+05     0.126520           26            2
```

Comment:
Do you need to add the `sample_one_particle.mrcs` file or am I misunderstanding?